test/env/env.go

```diff
@@ -0,0 +1,53 @@
+package env
+
+import (
+	. "github.com/onsi/gomega"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/api/v1alpha1"
+	nmstatev1beta1 "github.com/nmstate/kubernetes-nmstate/api/v1beta1"
+	"github.com/nmstate/kubernetes-nmstate/test/environment"
+)
+
+var (
+	cfg               *rest.Config
+	Client            client.Client         // You'll be using this client in your tests.
+	KubeClient        *kubernetes.Clientset // You'll be using this client in your tests.
```

Comment:
I can only see the Client (line above) to be used. What is the difference?

Reply:
First one is controller-runtime client it has different API and a cache, second one is client-go one does not have a cache, let me check if we are really not using it.

Reply:
It's used by the reporter 
```golang
func writePodsLogs(writer io.Writer, namespace string, sinceTime time.Time) {   
    podLogOpts := corev1.PodLogOptions{}                                        
    podLogOpts.SinceTime = &metav1.Time{Time: sinceTime}                        
    podList := &corev1.PodList{}                                                
    err := testenv.Client.List(context.TODO(), podList, &dynclient.ListOptions{})
    Expect(err).ToNot(HaveOccurred())                                           
    podsClientset := testenv.KubeClient.CoreV1().Pods(namespace)                
                                                                                
    for _, pod := range podList.Items {                                         
```